### PR TITLE
Exploring tool behaviour dictionary locale

### PR DESCRIPTION
**Pull request by 84634E1A607A**
**Brief Description of What This PR Does**

Added locale support for behaviour dictionary

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [ ] Initial code review passed (this and further items should not be checked by the PR author)
- [ ] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

File: locale/en.po [hunk 1]
```diff
@@ -1960,6 +1960,26 @@ msgstr "\"{0}\" - {1}"
 msgid "ART_BY"
 msgstr "Art by {0}"
 
+#: ../src/general/BehaviourDictionary.cs:102
+msgid "BEHAVIOUR_AGGRESSION"
+msgstr "Aggression"
```
Comment (hhyyrylainen) [inline locale/en.po:1965]: Don't these translations already exist for the GUI? This new feature should use the existing translation keys.

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]

File: locale/en.po (hunk 1 above)
Comment (84634E1A607A) [reply locale/en.po:1965]: Oh it should.

Comment (84634E1A607A) [reply locale/en.po:1965]: No they don't. The current ones are like

```gettext
#: ../src/microbe_stage/editor/BehaviourEditorSubComponent.tscn:141
msgid "OPPORTUNISTIC"
msgstr "Opportunistic"
```

Comment (hhyyrylainen) [reply locale/en.po:1965]: I guess then we need these new translations.

**Review by hhyyrylainen — APPROVED** [review]

**Pull request merged**